x/leverage/keeper/keeper_test.go

```diff
@@ -1273,36 +1188,6 @@ func (s *IntegrationTestSuite) TestBorrowAmountInvariant() {
 	s.Require().False(broken)
 }
 
-func (s *IntegrationTestSuite) TestBorrowAPYInvariant() {
-	s.app.LeverageKeeper.SetBorrowAPY(s.ctx, umeeapp.BondDenom, sdk.NewDec(2))
-
-	// check invariant
-	_, broken := keeper.BorrowAPYInvariant(s.app.LeverageKeeper)(s.ctx)
-	s.Require().False(broken)
-
-	// sets the borrow APY to 0
-	s.app.LeverageKeeper.SetBorrowAPY(s.ctx, umeeapp.BondDenom, sdk.NewDec(0))
-
-	// check invariant
-	_, broken = keeper.BorrowAPYInvariant(s.app.LeverageKeeper)(s.ctx)
-	s.Require().False(broken)
-}
-
-func (s *IntegrationTestSuite) TestLendAPYInvariant() {
```

Comment:
Note: Removed test as manually setting lend APY is no longer possible (always derived instantly)